pytket/tests/mapping_test.py

```diff
@@ -209,8 +216,64 @@ def test_basic_mapping() -> None:
     assert len(circ.get_commands()) == 10
 
 
+def test_MultiGateReorderRoutingMethod() -> None:
+    circ = Circuit(5)
+    arc = Architecture([[0, 1], [1, 2], [2, 3], [3, 4]])
+    # Invalid opration
+    circ.CZ(0, 2)
+    # Valid operations that can all be commuted to the front
+    circ.CZ(0, 1)
+    circ.CZ(1, 2)
+    circ.CZ(3, 2)
+    circ.CX(3, 4)
+
+    init_map = dict()
+    init_map[Qubit(0)] = Node(0)
+    init_map[Qubit(1)] = Node(1)
+    init_map[Qubit(2)] = Node(2)
+    init_map[Qubit(3)] = Node(3)
+    init_map[Qubit(4)] = Node(4)
+    pl = Placement(arc)
+    pl.place_with_map(circ, init_map)
+    # LexiRouteRoutingMethod should insert exactly one SWAP to route the final CZ gate
+    MappingManager(arc).route_circuit(
+        circ, [MultiGateReorderRoutingMethod(10, 10), LexiRouteRoutingMethod(50)]
+    )
+    assert circ.valid_connectivity(arc, False)
```

Comment:
could you change this to: assert circ.valid_connectivity(arc, allow_bridge=False) or similar to make clear what the parameter is doing here